kernel/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ add_library(Logger SHARED Logger.cpp)
 set_target_properties(Logger PROPERTIES POSITION_INDEPENDENT_CODE true)
 
 if( hpGEM_USE_MPI )
-  target_link_libraries(Logger "${MPI_LIBRARIES}")
+  target_link_libraries(Logger PRIVATE MPI::MPI_CXX)
```

Comment:
Might this also be necessary for the other kernel targets? (Might have forgotten this in the dependency graph)

Reply:
I think the Logger does not care about the rest, but I do not know.